code/vza/vza.js

```diff
@@ -754,57 +819,132 @@ $(document).on("knack-view-render.view_466", function (event, view, data) {
-  var recordsTable = `
-    <div class="assignments-table">
-      <table class="kn-table kn-table-table is-bordered">
-        <thead>
-          <tr>
-            <th>
-              <span class="table-fixed-label">
-                <span>Time</span>
-              </span>
-            </th>
-            <th>
-              <span class="table-fixed-label">
-                <span>Sector</span>
-              </span>
-            </th>
-            <th>
-              <span class="table-fixed-label">
-                <span>Location</span>
-              </span>
-            </th>
-          </tr>
-        </thead>
-        <tbody id="shift-table-body">
-        </tbody>
-      </table>
-    </div>`;
+// #### Sign Up page ####
+// Hide Knack generated "Available Assignments" table, create and add table that condenses
+// sign up for multiple officer_assignments into one button
+$(document).on("knack-view-render.view_466", function (event, view, data) {
+  var tableConfig = {
+    columns: {
+      // Header Title: Getter() for data in record
+      Time: function (record) {
+        return record[fields.timeField];
+      },
+      Sector: function (record) {
+        return record[fields.locationFieldRaw][0].identifier.split(" - ")[0];
+      },
+      Location: function (record) {
+        return record[fields.locationFieldRaw][0].identifier.split(" - ")[2];
+      }
+    },
+    addTimeFilters: true,
+    isCancelOnly: false
+  };
+
+  var recordsTable = createRecordsTable(view.key, tableConfig);
 
   // Append table, then request records and append shifts to table body
   $("#" + appSpecifics.availableAssignmentsView + "> div.view-header")
     .after(recordsTable)
     .ready(function () {
-      requestRecords(filters.all);
+      requestRecords(filters.all, view.key, tableConfig);
     });
 });
 
-// Assignment Details page
-// Update start button text
-$(document).on("knack-view-render.view_449", function (event, view, data) {
-  $("#view_449 button strong")[0].innerText = " Start Assignment";
+// #### My Assignments page ####
+// Hide Knack generated "Today's Assignments" and "Future Assignments" table, create and add tables that condense
+// cancellation for multiple officer_assignments into one button
+
+// Helper to create folder icon link in custom table data cells
+function addRecordLinkToTableConfig(view, config) {
+  var slug = view.scene.slug;
+
+  function createFolderIconLink(record) {
+    return `
+    <a href="#${slug}/assignment-details/${record.id}" 
+    class="kn-link kn-link-page" address="true" style="text-align: center; ">
+      <span class="level is-compact">
+        <span class="icon is-left">
+          <i class="fa fa-folder-open-o"></i>
+        </span>
+      </span>
+    </a>
+    `;
+  }
+
+  return {
+    ...config,
+    columns: { Details: createFolderIconLink, ...config.columns }
+  };
+}
+
+$(document).on("knack-view-render.view_447", function (event, view, data) {
```

Comment:
did we want to use a config reference for this view?

Reply:
Yes, I will include these changes in my latest PR. 🙏